resources/puppetlabs/lein-ezbake/template/foss/ext/default.erb

```diff
@@ -2,8 +2,8 @@
 # Init settings for <%= EZBake::Config[:project] %>
 ###########################################
 
-# Location of your Java binary (version 8)
-JAVA_BIN="/usr/bin/java"
+# Location of your Java binary
+#JAVA_BIN=""
```

Comment:
I found out that the `puppetserver` bin (which I can't find the source of now) also reads the defaults file. If that needs `JAVA_BIN` then this change is breaking.